README.md

```diff
@@ -51,36 +45,20 @@ Our team wishes to experiment with the idea of an installer in a more structured
 
 So far, our explorations have been quite fruitful, so we wanted to share and keep exploring.
 
-## Building
-
-Since you'll be using `nix-installer` to install Nix on systems without Nix, the default build is a static binary.
-
-Build it on a system with Nix:
-
-```bash
-nix build github:determinatesystems/nix-installer
-```
-
-Then copy the `result/bin/nix-installer` to the machine you wish to run it on.
-
-If you don't have Nix, yet still want to contribute, you can also run `cargo build` like a normal Rust crate.
-
-## Installing
+## Usage
 
 Install Nix with the default planner and options:
 
 ```bash
-./nix-installer install
```

Comment:
It feels kinda meh to me to recommend downloading the installer every time they want to do anything with it... Maybe we could keep this change, but in further invocations recommend downloading it somewhere and running that if they want to explore the options, etc.

Reply:
Ya agree!